client/cli/src/runner.rs

```diff
@@ -195,7 +195,11 @@ impl<C: SubstrateCli> Runner<C> {
 }
 
 /// Log information about the node itself.
-pub fn print_node_infos<C: SubstrateCli>(config: &Configuration) {
+/// (and hint to people who haven't set up logging how to do it)
+pub fn print_node_infos<C: SubstrateCli>(config: &Configuration) {	
+	if log::max_level() == log::LevelFilter::Off {
```

Comment:
But in order for substrate to stfu you have to actively set it to `-loff`, no? So if the user knows how to do that, doesn't that imply they know how the logging flag works?

Reply:
The `temp_base_path_works` test for example fails on my machine because the created process didn't produce any log output unless I add in `.env("RUST_LOG", "info")`. (In the less-sleeps PR I've added that into the test). It does seem possible to not have logging configured.

Reply:
I'm here with @dvdplm. 

The test is using `Stdio::piped()` that will result in not printing anything.

Reply:
Cool - will close this for now. If I see it again will dig into exactly why it thought it shouldn't produce any logging.